tools/cli/datalab.py

```diff
@@ -170,6 +192,12 @@ def run():
                 dest='zone',
                 default=None,
                 help=_ZONE_HELP)
+        if command_config['require-email']:
+            subcommand_parser.add_argument(
+                '--email',
```

Comment:
I think my comment still holds ... about this being worded as email in the flags, as opposed to git-user or something that indicates use.